### PR TITLE
Fixed #27162 -- Moved CreateExtension migration operation to core

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: docs/ref/databases.txt [hunk 1]
```diff
@@ -164,6 +164,14 @@ lookups that use the ``LIKE`` operator in their SQL, as is done with the
 
 .. _PostgreSQL operator class: http://www.postgresql.org/docs/current/static/indexes-opclass.html
 
+Migration operation for adding extensions
+-----------------------------------------
+
+If you need to add a PostgreSQL extension (like hstore, postgis, etc.) through
```
Comment (timgraham) [inline docs/ref/databases.txt]: Add double backticks around hstore and postgis.
through -> using

File: docs/ref/databases.txt [hunk 2]
```diff
@@ -164,6 +164,14 @@ lookups that use the ``LIKE`` operator in their SQL, as is done with the
...
+a migration, you can use the
```
Comment (timgraham) [inline docs/ref/databases.txt]: chop "you can"

File: docs/ref/contrib/gis/install/postgis.txt [hunk 3]
```diff
@@ -65,7 +65,16 @@ functionality::
...
 The database user must be a superuser in order to run
 ``CREATE EXTENSION postgis;``. The command is run during the :djadmin:`migrate`
-process.
+process. An alternative is to use a migration operation in your project::
+
+    from django.db import migrations
+    from django.db.backends.postgresql.migrations.operations import CreateExtension
+
+    class Migration(migrations.Migration):
+
```
Comment (timgraham) [inline docs/ref/contrib/gis/install/postgis.txt]: Add "..." to show this may not be a complete migration?

**Comment by claudep** [comment]
Docs part committed in 6b54504

**Pull request closed without merging**